src/main/java/cz/mzk/rest/CustomSolrClient.java

```diff
@@ -89,6 +90,18 @@ public long getNumFound(final SolrQuery queryParams) {
         return numFound;
     }
 
+    public void sendSolrInputDocument(SolrInputDocument solrInputDoc)  {
+        try {
+            solrClient.add(solrInputDoc);
+        } catch (SolrServerException e) {
+            log.warn("Can't added SolrInputDocument to the custom Solr client! " + e.getMessage());
```

Comment:
pro obe dve situace muzes vypsat stejny error message, neco jako "Can't index a document to the Solr instance"
pridavas dokument do instanci Solru, ne klientu :)